src/com/vivian/java/leetcode/CCI3_6AscendingStack.java

```diff
@@ -0,0 +1,31 @@
+package com.vivian.java.leetcode;
+import com.vivian.java.collections.Stack;
+/* Write a program to sort a stack in ascending order(with biggest items on top).
+You may use at most one additional stack to hold items,
+but you may not copy the elements into any other data structure.
+Tht stack supports  supports the following operations:push,pop,peek,and isEmpty.
+
+claritification: ascending ording such as 54321
```

Comment:
The clarification should be top or bottom of stack holds the minimum value?

Reply:
done